deploy/registration-service/registration-service.yaml

```
@@ -113,10 +114,20 @@ objects:
                   path: /api/v1/health
                   port: 8080
                   scheme: HTTP
-                initialDelaySeconds: 30
+                initialDelaySeconds: 1
                 periodSeconds: 1
                 successThreshold: 1
                 timeoutSeconds: 1
+              startupProbe:
+                failureThreshold: 18
+                httpGet:
+                  path: /api/v1/health
+                  port: 8080
+                  scheme: HTTP
+                initialDelaySeconds: 1
+                periodSeconds: 10
+                successThreshold: 1
+                timeoutSeconds: 1
```

It means the prob will be called 18 times (after one second initial delay) every 10 seconds before failing. So it's 1, 10, 11,  21, ... 181 seconds before failing. Correct?

If so, wouldn't it be better to have smaller intervals, something like:
```suggestion
              startupProbe:
                failureThreshold: 60
                httpGet:
                  path: /api/v1/health
                  port: 8080
                  scheme: HTTP
                initialDelaySeconds: 1
                periodSeconds: 3
                successThreshold: 1
                timeoutSeconds: 1
```

---

Good point, since this is only for startup we can have a shorter intervals. 👍